pkg/processor/runtime/python/py/requirements/python3_12.txt

```diff
@@ -0,0 +1,2 @@
+-r common.txt
+msgpack==1.0.8
```

Comment:
Can you try using the latest version 1.1.0 - https://pypi.org/project/msgpack/
If we can bump it now it will be great.